app/services/force/short_form_service.rb

```diff
@@ -61,6 +61,7 @@ def self.autofill_reset(application, listing_id)
         name: nil,
         agreeToTerms: nil,
         shortFormPreferences: [],
+        alternateContact: { appMemberId: nil },
```

Comment:
can we do the same thing with other application members? I think we want new ones for every application

Reply:
nevermind, seems like we're already OK.